.env

```diff
@@ -0,0 +1 @@
+VITE_MOVIE_API_KEY=e85d401facf2409590d6110e059c9c79
```

Comment:
This token shouldn't be public as it could be used be anyone when exposed on github.  Add your .env to .gitignore before pushing to github next time.